examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -120,41 +128,51 @@ typedef Particles<ParticleDescription<
 struct Hydrogen
 {
     static const float_X numberOfProtons  = 1.0;
-    static const float_X numberOfNeutrons = 0.0;
+    static const float_X numberOfNeutrons = 1.0;
```

Comment:
This would be [Deuterium](https://en.wikipedia.org/wiki/Deuterium) 
I think you wanted a hydrogen gas molecule, having 2 protons and no neutron.